includes/core-changes/networking/5.0/negotiatestream-sslstream-dont-fail-on-successive-begin-calls.md

```diff
@@ -0,0 +1,41 @@
+### NegotiateStream and SslStream allow successive Begin operations
+
+Error cases on security streams are handled differently and successive calls to `BeginAuthenticateAsServer` or `BeginAuthenticateAsClient` no longer fail.
```

Comment:
per the discussion in the original notes, this should be something like "may no longer fail" The exclusiveness is now enforced by the underlying task instead of the Begin/End pair so  there is some reface condition. It is still generally not OK to call Begin multiple times without calling End.